larger_than_memory.ipynb

```diff
@@ -0,0 +1,714 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    adata.layers["raw"] = adata.X.copy()  # Making sure we keep access to the raw counts
</pre><p>We don't have raw counts!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/ehrapy-tutorials/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>